src/bin/vip-import.js

```diff
@@ -18,3 +18,12 @@ command( )
 	.argv( process.argv, async () => {
 		await trackEvent( 'vip_import_command_execute' );
 	} );
+
+command( )
+	.command( 'media', 'Import media files to your application from a compressed archive' )
+	.command( 'media-progress', 'Import media files to your application from a compressed archive' )
+	.command( 'validate-files', 'Validate your media file library' )
```

Comment:
So I think this will have to be regression tested and removed from the sql command above:
https://github.com/Automattic/vip/blob/9e1ebb3d0c2047f6382043a9a84bc0716bfd8c4c/src/bin/vip-import.js#L16-L16

Reply:
Maybe even just `validate` ? Because it now lives under the `media` subcommand

Reply:
Or for the sake of conserving time, let's remove it from here and leave it where it was before. We can punt this into a story for a future sprint maybe.